tests/Benchmarks/Microsoft.Experimental.Collections/RefDictionaryPerformanceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+
+namespace Microsoft.Collections.Extensions.Tests
+{
+    [MemoryDiagnoser]
+    public class RefDictionaryPerformanceTests
+    {
+        [Params(100000)]
```

Comment:
Nit: on large numbers like this we use `_` to make it easier to read, i.e `100_000`